desktop/apps/auction/routes.js

```diff
@@ -48,11 +49,31 @@ export async function index (req, res, next) {
     // TODO: Refactor out Backbone
     const auctionModel = new Auction(sale)
 
+    /**
+     * An e-commerce sale is one which is created by following these steps:
+     * https://github.com/artsy/auctions/blob/master/auctions_support_faq.md#how-do-i-create-an-e-commerce-sale
+     * It enables relations between one or more conventional auctions and
+     * represents "Buy Now"-like artworks
+     */
+    const isEcommerceSale = !sale.is_auction && isObject(sale.promoted_sale)
+
+    // If an e-commerce sale, remove all sort options that are Auction related
+    let artworkBrowserSortOptions = auctionWorksInitialState.sortMap
+    let sort = auctionWorksInitialState.filterParams.sort
+    if (isEcommerceSale) {
```

Comment:
@sweir27 - ah, your comment got overwritten via a rebase. Reposting:

> Can we make this change in the initialState instead of here?

Looking into this, unfortunately not. We need access to the sale state from MP in order to determine this value and update filters, which would require us to partially apply our app reducers and loose a lot of our static export definitions. I'm sure we could make this a bit cleaner but would rather save for another cleanup PR.